ui/app/routes/dynamic_evaluations/projects/$project_name/DynamicEvalRunSelector.tsx

```diff
@@ -114,6 +136,8 @@ export function DynamicEvalRunSelector({
               role="combobox"
               aria-expanded={open}
               className="flex w-96 items-center justify-between gap-2"
+              onFocus={loadInitialRuns}
```

Comment:
Moving this outside of an effect to lazily run the initial query only when the user first interacts with the dropdown trigger.